docs/usage/self-hosted-experimental.md

```diff
@@ -14,6 +14,10 @@ We do not follow Semantic Versioning for any experimental variables.
 These variables may be removed or have their behavior changed in **any** version.
 We will try to keep breakage to a minimum, but make no guarantees that an experimental variable will keep working.
 
+## `RENOVATE_CACHE_DECORATOR_MINUTES`
```

Comment:
Do we definitely need a new option, and cannot reuse the one used in npm? Or is this for opting into this feature in the first place?

Reply:
I've implemented it according to [this comment](https://github.com/renovatebot/renovate/pull/20795#discussion_r1129683991), but personally I don't think we need env-based option at all, since many of functions are decorated with their own reasonable TTL values.

Reply:
the npm env changes the behavior och soft and hard timeout.

if not set, use normal TTL value as hard limit lie now. if set use env value as hard limit and ttl as soft timeout

Reply:
The maximum `ttlMinutes` we have is 4 weeks, the minimum is 10 minutes

Reply:
> if not set, use normal TTL value as hard limit lie now. if set use env value as hard limit and ttl as soft timeout

Are we okay to use `RENOVATE_CACHE_DECORATOR_MINUTES` for setting hard TTLs, while `RENOVATE_CACHE_NPM_MINUTES` seems to be defining the soft ones?

Reply:
I can do whatever you want here, I'm just not completely understanding the end goal and therefore I'm asking

Reply:
@viceice can we reuse the same config used for npm?

Reply:
sure, we probably should rename it then?

Reply:
- `RENOVATE_CACHE_DECORATOR_MINUTES` is the "soft TTL" for NPM
- `GlobalConfig` contains "hard TTL", and its name doesn't relate directly to NPM
- Unlike NPM case, lots of functions set their own "soft TTL", and I'm not sure we want too override all of them with a single value
- If use "hard TTL" from `GlobalConfig` or any env var, it's useful to know that maximum TTL value for decorator I was able to find is 4 weeks

Reply:
what about soft-ttl plus hard-ttl?

Reply:
I think datasources will still set their own soft TTL but a system setting for hard TTL makes sense

Reply:
> what about soft-ttl plus hard-ttl?

I like this idea, let's do it this way? @rarkins

Reply:
Yes but do we still need an env variable? We have https://github.com/renovatebot/renovate/blob/5d82120bbbdd922cbd451eba1d981ae0d5e70fc8/lib/config/options/index.ts#L2278-L2286

Reply:
Oh yes, we don't need it now